### ISSUE TITLE
Cross entropy

### ISSUE THREAD

**Issue by adam-r-kowalski** [adam-r-kowalski/compute_graph#58]
Implement the cross entropy loss function to measure distance between two probability distributions.

https://machinelearningmastery.com/cross-entropy-for-machine-learning/
